components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/utils/JWTAuthenticationUtils.java

```diff
@@ -34,6 +34,10 @@ public class JWTAuthenticationUtils {
     public static Message addToMessageContext(Message inMessage, HashMap<String,Object> authContext) {
         for (String key : authContext.keySet()) {
             inMessage.put(key, authContext.get(key));
+            // Exchanging the scopes to message context
+            if (key.equals("user_rest_api_scopes")) {
```

Comment:
Lets use a proper constant. There's already defined one too, we can try to reuse

Reply:
Addressed via : https://github.com/wso2/carbon-apimgt/pull/12038/commits/054b4b3c69d9059612489c10b99a58070e3dd586